src/main/java/frc/robot/Constants.java

```diff
@@ -305,15 +305,15 @@ public static final class TunerConstants {
         private static final Slot0Configs steerGains =
                 new Slot0Configs()
                         .withKP(150)
-                        .withKI(0)
+                        .withKI(50)
                         .withKD(0.2)
                         .withKS(0.32)
                         .withKV(1.5)
                         .withKA(0);
         // When using closed-loop control, the drive motor uses the control
         // output type specified by SwerveModuleConstants.DriveMotorClosedLoopOutput
         private static final Slot0Configs driveGains =
-                new Slot0Configs().withKP(5).withKI(0).withKD(0).withKS(0).withKV(0).withKA(0);
+                new Slot0Configs().withKP(5).withKI(8).withKD(0).withKS(0).withKV(0).withKA(0);
```

Comment:
We should definitely tune kS for our drive gains

Reply:
Yeah, I was thinking about that after reading the CHS discord and seeing other people use FF.